### ISSUE TITLE
Improve visibility of bus lanes to reduce the number of vehicles traveling down the wrong lane.

### ISSUE THREAD

**Issue by akozlik** [cforlando/2016-project-proposals#2]
## Short description of the idea (140 character tweet):

Improve visibility of bus lanes to reduce the number of vehicles traveling down the wrong lane.
## What is your project concept or idea? What challenge or opportunity will it solve?

Driving downtown is really confusing. The current bus lane markings on the road are not enough to keep people from using them as driving lanes. Improved signage that a driver can read should reduce the number of vehicles driving where they don't belong.
### Do you have photos of them? Or quotes?

Lynx/Lymmo drivers will benefit from this project. It will also keep drivers downtown from receiving tickets or causing accidents by driving the wrong lane.
## Issue area(s) relevant to your project idea:
- Mobility/Transportation
- Public Safety
## What is the current state of your project idea?

I have an idea or **concept** of a solution, but have not yet prototyped or tested it
### Do you have any links to news articles, text, mockups, code, or data?

I would like to see us discuss options with the City and possibly interview Lymmo drivers. We need to index where the bus lanes are and where drivers make the most mistakes.
### Are you representing a group in submitting this project idea?

I am the Co-Captain for Code for Orlando. My technical background is in both web and mobile development. This challenge is important to me because I've personally seen several people almost get in accidents because they're driving in the wrong lane.

**Comment by devdupont** [comment]
I like the idea of luminescent paint or colorful, solar-powered, side columns at bus lane intersections.

Something that draws the eye and is not stuck to the ground. Drivers don't actually look down at the road

**Comment by akozlik** [comment]
Yeah I was thinking just plain street signs since they're cheap and feasible. We can use the metal wrappers around poles so they can be removed if necessary as well.

**Comment by kelvint63** [comment]
I did a little feature about that on Facebook a year ago. I'm surprised there hasn't been a serious accident at any of the intersections heading south on Magnolia (the wrong way); just because there hasn't been one doesn't mean there's no problem? The longer it goes the more the chances of a major wreck, when more people do it and repeat it because they feel the risk are low... until the Perfect Storm and lost of life! 
A bit over-dramatic but it seams that nothing will be done until there is major loss-of-life!
Here's the FB link:  https://www.facebook.com/kelvin.thompson.jr/posts/10152676181086498
